<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="basicSettings_horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="onlySettings_verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="serialPort_horizontalLayout">
         <item>
          <widget class="QLabel" name="serialPortLabel">
           <property name="text">
            <string>Serial Port:</string>
           </property>
           <property name="buddy">
            <cstring>serialPortsComboBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="serialPortsComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="baudRate_horizontalLayout">
         <item>
          <widget class="QLabel" name="baudRateLabel">
           <property name="text">
            <string>Baud Rate:</string>
           </property>
           <property name="buddy">
            <cstring>baudRatesComboBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="baudRatesComboBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="openSerialButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>471</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>libs/GraphPlot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
